docker-compose.demo.yml

```diff
@@ -26,7 +26,7 @@ services:
 
 
   mysql:
-    image: islandoracollabgroup/isle-mysql:1.5.4
+    image: borndigital/isle-mysql:1.5.5
```

Comment:
I assume these "borndigital" refs are temporary

Reply:
Yes just for passing the first series of tests. They'll be reverted for the release.